tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf

```diff
@@ -0,0 +1,23 @@
+body common control
+{
+  bundlesequence => { "test", "check" };
+}
+
+bundle agent test
+{
+  meta:
+    "description" -> { "CFE-3569" }
+      string => "Make sure sys.os_name_human does not resolve to 'Unknown'";
+}
+
+bundle agent check
+{
+  classes:
+      "passed" expression => not(strcmp("$(sys.os_name_human)", "Unknown"));
```

Comment:
Can you also test that it is not empty string `""` and does not contain `os_name_human`?